common/tools/dev-tool/src/commands/admin/migrate-package.ts

```diff
@@ -291,12 +347,13 @@ function setScriptsSection(scripts: PackageJson["scripts"]): void {
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 function setFilesSection(packageJson: any): void {
-  packageJson.files = ["dist/", "README.md", "LICENSE"];
+  packageJson.files = ["dist/", "README.md", "LICENSE", "review/", "CHANGELOG.md"];
```

Comment:
Could we limit this to management packages? They are needed for some automation, but not beneficial to data plane packages.

Reply:
Fixed